docs/pages/o11y/logging/central_hpm.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Centralized Hostpath Mapper
+sidebar_label: Centralized Hostpath Mapper [PRO]
+---
+
+This feature is an extention to the existing [hostpath mapper](/docs/o11y/logging/hpm) component of vcluster.
+Currently when enabled, hostpath mapper can support the following usecases:
+1. [Enabling container based logging used by tools like fluentd, logstash etc.](/docs/o11y/logging/elk_stack) inside vcluster
+2. [Enabling pod based logging used by loki](/docs/o11y/logging/grafana_loki) inside vcluster
+3. [Velero restic backups] inside vcluster
```

Comment:
@neogopher this is empty since we have not yet accomodated the backup section in the new structure. Lets not forget about this